multimedia/csfml/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                csfml
+version             2.5.1
+revision            0
+categories          multimedia devel
+maintainers         nomaintainer
+license             zlib
+
+description         C bindings for SFML, the Simple and Fast Multimedia Library
+
+long_description    ${description}. It provides a simple interface to the various \
+                    components of your computer, to ease the development of games and \
+                    multimedia applications. It is composed of five modules: system, \
+                    window, graphics, audio and network.
+
+homepage            https://www.sfml-dev.org/
+master_sites        https://www.sfml-dev.org/files/
```

Also a bit of a nitpick, but you can use `${homepage}` to not repeat yourself in `master_sites`:

```suggestion
master_sites        ${homepage}/files/
```